### ISSUE TITLE
Daybreak: Change command APC fail condition.

### ISSUE THREAD

**Issue by Superxpdude** [Superxpdude/tmtm_operations#1]
Replace the current fail condition (command APC loss = mission failed) with a respawn disabled scenario.
Instead, end the mission if the command APC gets destroyed and either all players are dead, or the device is activated.
Activate the device once the command APC gets destroyed to put pressure on the players.

**Comment by Superxpdude** [comment]
Fixed in 509875c
